tests/unit/backend/helpers/jest-db.ts

```diff
@@ -269,6 +273,58 @@ const getFullFormById = async (
 ): Promise<IPopulatedForm | null> =>
   await getFormModel(mongoose).getFullFormById(formId)
 
+const insertFormSubmission = async ({
+  formId,
+  submissionType = 'encryptSubmission',
+  version = '1',
+  encryptedContent = 'encryptedContent',
+}: {
+  formId?: ObjectID
+  submissionType?: string
+  version?: string
+  encryptedContent?: string
+} = {}): Promise<{
+  submission: ISubmissionSchema
+}> => {
+  const SubmissionFormModel = getSubmissionModel(mongoose)
```

Comment:
the `Form` in the name isn't necessary, plus it makes sense for `getSubmissionModel` to return something called `SubmissionModel` haha
```suggestion
  const SubmissionModel = getSubmissionModel(mongoose)
```